src/app.ts

```diff
@@ -45,10 +55,30 @@ async function app() {
 
   logger.info(`Running SUSHI ${getVersion()}`);
 
-  // Check that input folder is specified
+  // If no input folder is specified, set default to current directory
   if (!input) {
-    logger.error('Missing path to FSH definition folder.');
-    program.help();
+    input = path.join('.');
+    logger.info('path-to-fsh-defs defaulted to current working directory');
+  }
+
+  // If a fsh subdirectory is detected, we are in an IG Publisher context
+  const isUsingFshDir = path.parse(input).base === 'fsh';
+
+  const fshSubdirectoryPath = path.join(input, 'fsh');
+  const hasFshDir = fs.existsSync(fshSubdirectoryPath);
+  const shouldUseFshDir = hasFshDir && input !== fshSubdirectoryPath;
```

Comment:
Given `const fshSubdirectoryPath = path.join(input, 'fsh');`, how is it ever possible for `input !== fshSubdirectoryPath` to return anything but `true`?

Reply:
Somehow I was thinking if it existed but wasn't used, but I don't think that's right anymore. But I'm also going to just re-evaluate this whole section and potentially change things around.